src/main/java/teammates/common/datatransfer/attributes/Gender.java

```diff
@@ -0,0 +1,26 @@
+package teammates.common.datatransfer.attributes;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Represents the gender of a student.
+ */
+public enum Gender {
+    @SerializedName("male")
```

Comment:
Do you really need this?

Reply:
Hmm some tests were failing locally, but they seem to work fine on Travis. I'll remove it.